swap_meet/vendor.py

```diff
@@ -1,2 +1,77 @@
 class Vendor:
-    pass
+    def __init__(self,inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+        # if inventory != None else inventory = []
```

Comment:
Your comment had the right idea! We can refactor using a ternary:

```suggestion
      self.inventory = inventory if inventory is not None else []
```